python/foglamp/plugins/storage/postgres/backup_restore/backup_postgres.py

```diff
@@ -47,37 +56,273 @@
 }
 """ Messages used for Information, Warning and Error notice """
 
-_logger = {}
-
 
 class Backup(object):
-    """ Backup API integration
+    """ Provides external functionality/integration for Backup operations
 
-    # CURRENTLY NOT IMPLEMENTED !!
+        the constructor expects to receive a reference to a StorageClient object to being able to access
+        the Storage Layer
     """
 
-    # noinspection PyUnusedLocal
-    def get_backup_list(self, limit, skip, status):
-        """  Retrieves backups information
+    _logger = None
+
+    _SCHEDULE_BACKUP_ON_DEMAND = "fac8dae6-d8d1-11e7-9296-cec278b6b50a"
```

Comment:
We should aim to be able to run based on named schedules and not embed UUID's like this. Will raise a Jira for changing this the scheduler at some point in the future. In the meantime we will have to stick with this.,